src/xdocs/config_javadoc.xml

```diff
@@ -1210,6 +1210,13 @@ public boolean isSomething()
             <td><code>false</code></td>
             <td>5.1</td>
           </tr>
+          <tr>
+            <td>allowedAnnotations</td>
+            <td>List of annotations that allow missed documentation.</td>
+            <td><a href="property_types.html#stringSet">String Set</a></td>
+            <td><code>{}</code></td>
+            <td>8.14</td>
+          </tr>
```

Comment:
Please append some example of usage of new option for code  on spring annotations as described in issue.
I think it is reasonable to make same valuable default value for some jdk annotations like https://docs.oracle.com/en/java/javase/11/docs/api/java.compiler/javax/annotation/processing/Generated.html or ...

Reply:
Hi @romani ,
I did the following changes:
- changed "since" to 8.15 as I see you've already released 8.14
- made `@Generated` the default value
- added an example in the documentation for a few Spring annotations
- adjusted unit tests to verify the new default behavior regarding `@Generated`
- rebased on top of master, squashed in a single commit